Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3469,6 +3469,20 @@ HiddenPageDOMTimerThrottlingEnabled:
     WebCore:
       default: false
 
+HiddenUntilFoundEnabled:
+   type: bool
+   status: testable
+   category: html
+   humanReadableName: "hidden=until-found"
```

Comment:
Don't we already support this? This is only hidden getter/setter support for it right?

Assuming we already support this feature I'd enable this by default as this is a trivial change to the getter/setter. But we should change the name here then to make it more clear this is solely about the getter/setter.

Reply:
I don't think we do? There's some work around the reveal algorithm, which removes the `hidden` attribute when find-in-page reaches it, and emits a `beforematch` event.